### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by KohliDev**
CRL update and corrected Package manager readme formatting

### CODE REVIEW

File: packages/liotad/README.md [hunk 1]
```diff
@@ -1,23 +1,34 @@
-#Liota Package Manager
-Liota package manager consists of two parts: (1) A PackageThread that actually loads files, maintain global data structures and run package initialization/clean-up codes and (2) A PackageMessengerThread which listens on one or more specific communication channels, which is a named pipe for now, to provide with an interface for users and automated agents to send commands to package manager.
+## Liota Package Manager
+
+Liota package manager consists of two parts:
+
+(1) A PackageThread that actually loads files, maintain global data structures and run package initialization/clean-up codes and
```
Comment (GregBollella) [inline packages/liotad/README.md]: Instead of 'codes' it would be better to say either 'logic' or 'code'

File: README.md [hunk 2]
```diff
@@ -36,13 +36,14 @@ Liota supports plain old BSD sockets, WebSocket and MQTT communication protocols
 * Identity class encapsulates certificates or credentials related to a connection used at both DCC and Device side.
 * TLSConf class encapsulates parameters related to TLS configuration.
 * CRLs(Certificate Revocation Lists) check has been implemented: [VRIFY_CRL_CHECK_CHAIN](https://docs.python.org/2/library/ssl.html#ssl.VERIFY_CRL_CHECK_CHAIN) in this mode CRLs of all certificates in the peer cert chain are checked.
- In order to enable the CRLs set the 'crl_path' in '/etc/liota/liota.conf', it requires a valid path to CRL in PEM format (preferable directory '/usr/lib/liota'). If the check is required to be disabled 'crl_path' should be left blank.
+ In order to enable the CRLs set the 'crl_path' in '/etc/liota/liota.conf', it requires a valid path to CRL; if in case intermediate CA is used the CRL should include both Intermediate CRL and Root CRL in PEM format (preferable directory '/usr/lib/liota'). If the check is required to be disabled 'crl_path' should be left blank.
```
Comment (pmasrani) [inline README.md]: /etc/liota/ path is configurable, in which case, /etc/liota/liota.conf can be misleading.

File: README.md [hunk 3]
```diff
@@ -36,13 +36,14 @@ Liota supports plain old BSD sockets, WebSocket and MQTT communication protocols
...
+* Password based CA certificates are not supported.
```
Comment (pmasrani) [inline README.md]: Please raise a feature request for the same.

File: README.md (hunk 2 above)
Comment (pmasrani) [inline README.md]: Why do we have a preferable directory '/usr/lib/liota'?

File: config/liota.conf [hunk 4]
```diff
@@ -16,7 +16,7 @@ enable_reboot_getprop = False
 # The number of attempts to retry in order to read iotcc.json files
 iotcc_load_retry = 3
 # Configurable queue timeout in order to wait for the response messages from IOTCC
-iotcc_response_timeout = 300
+iotcc_response_timeout = 600
```
Comment (pmasrani) [inline config/liota.conf]: I believe this is part of a separate pull request. Please remove it from here.

File: README.md [hunk 5]
```diff
@@ -36,13 +36,14 @@ Liota supports plain old BSD sockets, WebSocket and MQTT communication protocols
...
 
 ## Device Discovery
-Liota SDK provides users a way to discover new devices at run-time through a dedicated discovery thread. The discovery thread could listen on a list of end points by spinning up a listening thread for each of them. Currently, Liota supports 4 kinds of end points: MQTT, COAP, Socket, and Named Pipe. However, because of security consideration on edge system, currently only MQTT and Named Pipe are allowed, especially Named Pipe is only accessible to its owner user.
+Under development: Liota SDK provides users a way to discover new devices at run-time through a dedicated discovery thread. The discovery thread could listen on a list of end points by spinning up a listening thread for each of them. Currently, Liota supports 4 kinds of end points: MQTT, COAP, Socket, and Named Pipe. However, because of security consideration on edge system, currently only MQTT and Named Pipe are allowed, especially Named Pipe is only accessible to its owner user.
```
Comment (pmasrani) [inline README.md]: Good! :)

File: packages/liotad/README.md [hunk 6]
```diff
@@ -1,23 +1,34 @@
...
-##PackageThread
+## PackageThread
 
 Currently, supported commands include package action commands and statistical commands.
 
-###Package action commands
+### Package action commands
 
-* **load** package_name sha1_checksum [package_name sha1_checksum] ...
+* **load** package_name <sha1_checksum> ...
```
Comment (pmasrani) [inline packages/liotad/README.md]: Might be useful to show how multiple packages can be loaded, just ... is not intuitive enough.

Something like:

**load** package_name <sha1_checksum> [(optional) package2_name <sha1_checksum_of_package2>...]

File: packages/liotad/README.md (hunk 1 above)
Comment (KohliDev) [reply packages/liotad/README.md]: Changed 'codes' to 'logic'

File: README.md (hunk 2 above)
Comment (KohliDev) [reply README.md]: I wanted to specify the specific parameter 'crl_path' which user needs to set hence mentioned the complete path.

Comment (KohliDev) [reply README.md]: '/usr/lib/liota' believe to exist on most of the gateways after the restart. And it is the preferred directory to keep the public certificates and public files for Liota.

File: README.md (hunk 3 above)
Comment (KohliDev) [reply README.md]: Sure.

File: README.md (hunk 5 above)
Comment (KohliDev) [reply README.md]: Thanks

File: config/liota.conf (hunk 4 above)
Comment (KohliDev) [reply config/liota.conf]: Raised a new PR 
https://github.com/vmware/liota/pull/206
addressing all the review comments.

I have closed this PR due to merge conflict.

File: packages/liotad/README.md (hunk 6 above)
Comment (KohliDev) [reply packages/liotad/README.md]: Done

**Pull request closed without merging**